src/zc/recipe/testrunner/README.txt

```diff
@@ -387,7 +387,7 @@ Running them with the current buildout will produce a failure:
     Running zope.testrunner.layer.UnitTests tests:
       Set up zope.testrunner.layer.UnitTests in 0.001 seconds.
       Running:
-     test (demo.tests.DemoTests) (0.000 s)
+     test (demo.tests.DemoTests) (... s)
```

Comment:
Huh, there's a renormalizer for `\d[.]\d+ seconds` to `0.001 seconds`, but no renormalizer for the `s` variant:

https://github.com/zopefoundation/zc.recipe.testrunner/blob/766796a13381d21a2b4bc24499ca93aae2d7e81a/src/zc/recipe/testrunner/tests.py#L45

I'd add a second renormalizer instead of using `...` in the doctest.